slade360/wrappers/start_visits.py

```diff
@@ -0,0 +1,164 @@
+from typing import Any, Dict, List, Literal, Optional, Union
```

Comment:
suggestion to use mypy to effectively help in type casting

Reply:
I was thinking we can have that reinforcement in future iterations